hooks/useFetchNodeReleasesData.ts

```diff
@@ -0,0 +1,47 @@
+import { useRouter } from 'next/router';
+import { useMemo } from 'react';
+import useSWR from 'swr';
+import { getNodeReleaseStatus } from '../util/nodeReleaseData';
+import type { NodeReleaseData } from '../types';
+
+const fetcher = (...args: Parameters<typeof fetch>) =>
```

Comment:
Maybe we can create a dedicated hook called `useJsonSWR` which already incorporates this fetcher. I can imagine this being used in more places....